tests/config.test.ts

```diff
@@ -3,16 +3,16 @@ import Romcal from '../lib';
 const { RomcalConfig, LiturgicalDayConfig } = Romcal;
 
 describe('getConfig()', () => {
-  test('should get general config if country doesnt have default configurations', async () => {
+  test('should get general config if country does’t have default configurations', async () => {
```

Comment:
I prefer using `'` in code comments, but not in localised strings (or any other string that is visible to the end user) I prefer using `’` (U+2019).

```suggestion
  test('should get general config if country doesn't have default configurations', async () => {
```

Reply:
I prefer to keep it that way here, so I don't have to use <code>`</code> or <code>"</code> here around the text description of this test.

Reply:
> I prefer to keep it that way here, so I don't have to use `` ` `` or `"` here around the text description of this test.

Pretty lazy you are! :rofl: I’d have no problem using either quote type, although I’d prefer usig `` ` ``.

That said, we can keep the typographic apostrophe, however, please do fix the typo. :wink:

Reply:
It was just to keep quoting type consistent here. But instead, I've another solution (or rather a workaround) :grinning::
```suggestion
  test('should get general config if country do not have default configurations', async () => {
```

Reply:
> It was just to keep quoting type consistent here.

Well, I personally don’t care about quoting type consistence. I prefer using single quote, then template string, then double quotes, however, when I need/want to using one of them in a string, I have no problem to use either of them. :man_shrugging:

> But instead, I've another solution (or rather a workaround) :grinning:

Go for it! :wink: